resources/sql/application-queries.sql

```diff
@@ -719,6 +744,29 @@ SELECT
 FROM application_hakukohde_reviews
 WHERE application_key = :application_key AND requirement = :requirement AND hakukohde = :hakukohde;
 
+-- name: yesql-upsert-attachment-hakukohde-review!
+INSERT INTO application_hakukohde_attachment_reviews (application_key, attachment_key, state, hakukohde)
+VALUES (:application_key, :attachment_key, :state, :hakukohde)
+ON CONFLICT (application_key, attachment_key, hakukohde)
+  WHERE hakukohde IS NOT NULL
```

Comment:
Tää lienee turha, `hakukohde` sarake on `NOT NULL`

Reply:
:+1: